deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -1115,8 +1133,14 @@ def callback_detect_integrity_state(line):
     return None
 
 
+def callback_start_synchronization(line):
+    if 'FIM sync module started' in line:
```

Comment:
This seems to have been added before the we started moving callbacks to  the new framework format. I have left the callback as It currently is, since it would require to refactor tests that are not related to this PR, making it larger than it already is.